File: PrApiTest/Model/Lease.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace PrApiTest.Model
{
    [Table("leases")]
    public class Lease
    {
        [Column("lease_id")]
        public int Id { get; set; }

        [Column("property_id")]
        public int PropertyId { get; set; }

        public virtual Property Property { get; set; }

        [Column("date_from")]
        public String DateFrom { get; set; }

        [Column("date_to")]
        public String DateTo { get; set; }

        [Column("amount")]
        public int AmountMonthly { get; set; }


    }
}

File: PrClient/src/app/models/lease.ts
import { Property } from "./property";
import { Client } from "./client";

export class Lease{
    id: number;
    propertyId: number;
    property: Property;
    dateFrom: Date;
    dateTo: Date;
    monthlyAmount: number;
    tenantId: number;
    tenant: Client;
    
}